ui/src/api/api.tsx

```diff
@@ -32,6 +32,23 @@ export const fetchDataSources = async (project: string) => {
     });
 };
 
+export const fetchDataSource = async (
+  project: string,
+  dataSourceId: string
+) => {
+  const axios = await authAxios(msalInstance);
+  return axios
+    .get<DataSource>(
+      `${getApiBaseUrl()}/projects/${project}/datasources/${dataSourceId}`,
```

Comment:
I think This new API is not supported in RBAC yet. Will be a problem. @blrchen  please suggest if we should include it in this PR or add it in another PR and merged in previous.

Reply:
@ahlag can you please make change in to enable this api on rbac as well, the change required is add a route for same api in rbac service https://github.com/feathr-ai/feathr/blob/main/registry/access_control/api.py

Reply:
@blrchen @Yuqing-cat 
Ok sure! How should I test RBAC? Checking the header in Network Tab?

Reply:
The proper way to test RBAC is to deploy a RBAC enabled Registry and then test with Feathr UI and Client.